src/dropdown/Dropdown.js

```diff
@@ -0,0 +1,188 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import {
+  View,
+  Text,
+  TouchableNativeFeedback,
+  TouchableOpacity,
+  ActivityIndicator,
+  Platform,
+  StyleSheet,
+  ViewPropTypes,
+  TextInput,
+  AppState,
+} from 'react-native';
+import Color from 'color';
+
+import { withTheme } from '../config';
+import { renderNode, nodeType, conditionalStyle, color } from '../helpers';
+import Icon from '../icons/Icon';
+import Input from '../../'
+import { abs } from 'react-native-reanimated';
+
+const defaultLoadingProps = (type, theme) => ({
+  color: type === 'solid' ? 'white' : theme.colors.primary,
+  size: 'small',
+});
+
+const Dropdown = (props) => {
+  const { data, onItemSelected, itemSelected, containerStyle, innerStyle, containerListStyle, listStyle } = props
+  let { indexSelected } = props
+  const defaultSelected = data[0]
+  const label = typeof defaultSelected.label !== 'undefined' ? defaultSelected.label : defaultSelected.value
```

Comment:
We could probably just check for the value if `undefined` is all we need
```suggestion
  const label = defaultSelected.label ? defaultSelected.label : defaultSelected.value
```